app/javascript/mastodon/features/account/components/header.js

```diff
@@ -328,12 +328,12 @@ class Header extends ImmutablePureComponent {
             )}
           </div>
 
-          <div className='account__header__tabs__name'>
+          <a className='account__header__tabs__name' href={`${account.get('url')}`} target='_blank' rel='noopener noreferrer'>
             <h1>
               <span dangerouslySetInnerHTML={displayNameHtml} /> {badge}
               <small>@{acct} {lockedIcon}</small>
             </h1>
-          </div>
+          </a>
```

Comment:
I don't know if I'd have that be a link.

Reply:
The reasoning behind this was to give users an easy way to open the remote profile without menu diving, but I'm open to alternative approaches too